selectrum.el

```diff
@@ -493,6 +493,13 @@ This is used to implement `selectrum-repeat'.")
 (defvar selectrum-active-p nil
   "Non-nil means Selectrum is currently active.")
 
+(defvar-local selectrum--skip-updates-p nil
+  "If selectrum should skip updates.
+
+In normal operation Selectrum checks for updating its UI after
+each command. When this variabele is non-nil the computation of
```

Comment:
```suggestion
each command. When this variable is non-nil the computation of
```

Reply:
Thanks I need to enable my spell checker again ;)